packages/schema/lib/schemas/ThrottleObjectSchema.js

```diff
@@ -36,6 +36,16 @@ module.exports = makeSchema(
           type: 'string',
         },
       },
+      retry: {
+        description:
+          'The effect of throttling on the tasks of the action. `true` means throttled tasks are automatically retried after some delay, while `false` means tasks are held without retry. It defaults to `true`.',
+        type: 'boolean',
+      },
+      filter: {
+        description: `EXPERIMENTAL: Account-based attribute to override the throttle by. You can set to one of the following: "free", "trial", "paid". Therefore, the throttle scope would be automatically set to "account" and ONLY the accounts based on the specified filter will have their requests throttled based on the throttle overrides while the rest are throttled based on the original configuration.`,
```

Comment:
1. Can you share why `filter` is experimental while `retry` is not?
2. Regarding "the throttle scope would be automatically set to 'account'", what would happen if `scope` is not `account`?

Reply:
>Regarding "the throttle scope would be automatically set to 'account'", what would happen if scope is not account?

`filter` is an account-based config setting that uses ONLY `account` as the scope and disregards any explicit scope set.

>Can you share why filter is experimental while retry is not?

I assumed we wanted to open up the `retry` to the public due to the need it was requested for.

```
As we're planning on increasing the polling rate for all integrations, we're concerned about one or two apps which might become rate limited and thus, want to throttle them in advance to avoid issues...

Ultimately I think we need to avoid the possibility of stacking up retry attempts faster than they'll actually be processed, causing a pile that only gets bigger with time...
```

from https://zapier.slack.com/archives/C04P5J5R8E4/p1716513877072559

Reply:
@FokkeZB Please, did you intend to make the root-level `retry` experimental when you suggested it? https://zapier.slack.com/archives/D3YRTGHAM/p1717433047352299

Reply:
What's effect of marking it as experimental? Does that imply it's only for internal usage? As the override-level `filter` property isn't experimental either, I don't see why the root-level one should.

Reply:
@FokkeZB Yes, experimental implies internal usage.  The override-level `filter` is experimental indirectly because the `override` object itself is experimental.

Reply:
Ah, ok. I don't have a strong opinion on whether this one should be experimental.